src/server/zset_family_test.cc

```diff
@@ -274,43 +274,43 @@ TEST_F(ZSetFamilyTest, ZUnion) {
   EXPECT_EQ(2, CheckedInt({"zadd", "z2", "3", "c", "2", "b"}));
   EXPECT_EQ(2, CheckedInt({"zadd", "z3", "1", "c", "1", "d"}));
 
-  resp = Run({"zunion", "z1", "z2", "z3", "weights", "1", "1", "k"});
+  resp = Run({"zunion", "3", "z1", "z2", "z3", "weights", "1", "1", "k"});
   EXPECT_THAT(resp, ErrArg("weight value is not a float"));
 
-  resp = Run({"zunion", "z1", "z2", "z3", "weights", "1", "1", "2", "aggregate", "something"});
+  resp = Run({"zunion", "3", "z1", "z2", "z3", "weights", "1", "1", "2", "aggregate", "something"});
   EXPECT_THAT(resp, ErrArg("syntax error"));
 
-  resp = Run({"zunion", "z1", "z2", "z3", "weights", "1", "2", "aggregate", "something"});
+  resp = Run({"zunion", "3", "z1", "z2", "z3", "weights", "1", "2", "aggregate", "something"});
   EXPECT_THAT(resp, ErrArg("weight value is not a float"));
 
-  resp = Run({"zunion", "z1", "z2", "z3", "aggregate", "sum", "somescore"});
+  resp = Run({"zunion", "3", "z1", "z2", "z3", "aggregate", "sum", "somescore"});
   EXPECT_THAT(resp, ErrArg("syntax error"));
 
-  resp = Run({"zunion", "z1", "z2", "z3", "withscores", "someargs"});
+  resp = Run({"zunion", "3", "z1", "z2", "z3", "withscores", "someargs"});
   EXPECT_THAT(resp, ErrArg("syntax error"));
 
-  resp = Run({"zunion", "z1", "z2", "z3"});
+  resp = Run({"zunion", "3", "z1", "z2", "z3"});
   EXPECT_THAT(resp.GetVec(), ElementsAre("a", "d", "c", "b"));
 
-  resp = Run({"zunion", "z1", "z2", "z3", "weights", "1", "1", "2"});
+  resp = Run({"zunion", "3", "z1", "z2", "z3", "weights", "1", "1", "2"});
   EXPECT_THAT(resp.GetVec(), ElementsAre("a", "d", "b", "c"));
 
-  resp = Run({"zunion", "z1", "z2", "z3", "weights", "1", "1", "2", "withscores"});
+  resp = Run({"zunion", "3", "z1", "z2", "z3", "weights", "1", "1", "2", "withscores"});
   EXPECT_THAT(resp.GetVec(), ElementsAre("a", "1", "d", "2", "b", "5", "c", "5"));
 
-  resp =
-      Run({"zunion", "z1", "z2", "z3", "weights", "1", "1", "2", "aggregate", "min", "withscores"});
+  resp = Run({"zunion", "3", "z1", "z2", "z3", "weights", "1", "1", "2", "aggregate", "min",
+              "withscores"});
   EXPECT_THAT(resp.GetVec(), ElementsAre("a", "1", "b", "2", "c", "2", "d", "2"));
 
-  resp =
-      Run({"zunion", "z1", "z2", "z3", "withscores", "weights", "1", "1", "2", "aggregate", "min"});
+  resp = Run({"zunion", "3", "z1", "z2", "z3", "withscores", "weights", "1", "1", "2", "aggregate",
+              "min"});
   EXPECT_THAT(resp.GetVec(), ElementsAre("a", "1", "b", "2", "c", "2", "d", "2"));
 
-  resp = Run({"zunion", "none1", "none2", "z3", "withscores", "weights", "1", "1", "2"});
+  resp = Run({"zunion", "3", "none1", "none2", "z3", "withscores", "weights", "1", "1", "2"});
   EXPECT_THAT(resp.GetVec(), ElementsAre("c", "2", "d", "2"));
 
-  resp =
-      Run({"zunion", "z1", "z2", "z3", "weights", "1", "1", "2", "aggregate", "max", "withscores"});
+  resp = Run({"zunion", "3", "z1", "z2", "z3", "weights", "1", "1", "2", "aggregate", "max",
```

Comment:
I see that we have only checks with 3 arguments, maybe change one of the cases to check different number

Reply:
LGTM